.github/workflows/integration-tests.yml

```diff
@@ -0,0 +1,57 @@
+name: Integration Tests
+
+on:
+    pull_request:
+        types: [opened, synchronize]
+
+    merge_group:
+
+jobs:
+    trigger-tests:
+        if: github.event_name == 'pull_request'
+        name: Trigger Tests
+        runs-on: ubuntu-latest
+        environment: "test-trigger-is"
+
+        steps:
+            - uses: actions/checkout@v3
+
+            - name: Generate GitHub App Token
+              id: generate-token
+              uses: actions/create-github-app-token@v1
+              with:
+                  app-id: ${{ secrets.DECO_WORKFLOW_TRIGGER_APP_ID }}
+                  private-key: ${{ secrets.DECO_WORKFLOW_TRIGGER_PRIVATE_KEY }}
+                  owner: ${{ secrets.ORG_NAME }}
+                  repositories: ${{secrets.REPO_NAME}}
+
+            - name: Trigger Workflow in Another Repo
+              env:
+                  GH_TOKEN: ${{ steps.generate-token.outputs.token }}
+              run: |
+                  gh workflow run vscode-isolated-pr.yml -R ${{ secrets.ORG_NAME }}/${{secrets.REPO_NAME}} \
```

Comment:
When we change the main branch back to main I'll need to update the vscode-isolated-pr.yml in the private repo, right?

Reply:
Oh, and let's merge this into bundle-integ branch, not main